account_move_line_reconcile_manual/models/account_reconcile_manual_model.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2023 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountReconcileManualModel(models.Model):
+    _name = "account.reconcile.manual.model"
+    _description = "Models for Manual Reconcile Write-off"
+    _check_company_auto = True
+    _rec_name = "ref"
+    _order = "sequence, id"
+
+    sequence = fields.Integer()
+    company_id = fields.Many2one("res.company", ondelete="cascade", required=True)
```

Comment:
I would expect an SQL index on this field

Reply:
done